docs/user/tutorials/os-quick-start.md

```diff
@@ -0,0 +1,194 @@
+2. Choose **Create**.
+3. In the created namespace, go to **Workloads > Deployemnts** and choose **Create**.
+1. Select the HTTPBin template.
+2. Choose **Create**.
+3. Go to **Configuration > Service Accounts** and choose **Create**. 
+4. Enter `httpbin` as your Service Account's name.
+5. Choose **Create**.
+6. Go to **Discovery and Network > Services** and choose **Create**. 
+7. Provide the following configuration details:
+    - **Name**: `httpbin`
+    - In the `Labels` section, add the following labels:
+      - **service**: `httpbin`
+      - **app**:`httpbin`
+    - In the `Selectors` section, add the following selector:
+      - **app**: `httpbin`
+    - In the `Ports` section, select **Add**. Then, use these values:
+      - **Name**: `http`
+      - **Protocol**: `TCP`
+      - **Port**: `8000`
+      - **Target Port**: `80`
+8. Choose **Create**.
+
+#### **kubectl**
+
+1. Create a namespace and export its value as an environment variable. Run:
+
+    ```bash
+    export NAMESPACE=api-gateway-tutorial
+    kubectl create ns $NAMESPACE
+    kubectl label namespace $NAMESPACE istio-injection=enabled --overwrite
+    ```
+
+2. Deploy a sample instance of the HTTPBin Service.
+
+    ```shell
+    cat <<EOF | kubectl -n $NAMESPACE apply -f -
+    apiVersion: v1
+    kind: ServiceAccount
+    metadata:
+      name: httpbin
+    ---
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: httpbin
+      labels:
+        app: httpbin
+        service: httpbin
+    spec:
+      ports:
+      - name: http
+        port: 8000
+        targetPort: 80
+      selector:
+        app: httpbin
+    ---
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: httpbin
+    spec:
+      replicas: 1
+      selector:
+        matchLabels:
+          app: httpbin
+          version: v1
+      template:
+        metadata:
+          labels:
+            app: httpbin
+            version: v1
+        spec:
+          serviceAccountName: httpbin
+          containers:
+          - image: docker.io/kennethreitz/httpbin
+            imagePullPolicy: IfNotPresent
+            name: httpbin
+            ports:
+            - containerPort: 80
+    EOF
+    ```
+
+    To verify if an instance of the HTTPBin Service is successfully created, run:
+
+    ```shell
+    kubectl get pods -l app=httpbin -n $NAMESPACE
+    ```
+
+    If successful, you get a result similar to this one:
+
+    ```shell
+    NAME                 READY    STATUS     RESTARTS    AGE
+    httpbin-{SUFFIX}     2/2      Running    0           96s
+    ```
+
+<!-- tabs:end -->
+
+## Expose a Workload
+
+<!-- tabs:start -->
+#### **Kyma Dashboard**
+
+1. In the `api-gateway-tutorial` namespace, go to **Discovery and Network > API Rules**.
+2. Choose **Create**.
+3. Provide the following details.
+  - **Name**: `httpbin`
+  - In the `Service` section, add:
+    - **Service Name**: `httpbin`
+    - **Port**: `8000`
+  - In the **Service** section, add:
```

Comment:
please dbl check cause if it's `Service`

Reply:
it's Gateway, thank you